### ISSUE TITLE
Seem to different at Leelaz network and Training network

### ISSUE THREAD

**Issue by CGLemon** [leela-zero/leela-zero#2587]
The Leelaz network consist of Resnet towers, policy head and value head. The all convolutional layer is no bias, but there is bias in weights form. The 'tfprocess.py'  also define  network struct. From 'tfprocess.py' ,we can kwon that batch normalization layer will be trained and the beta parameter will be written in convolutional bias.
see the detail at
https://github.com/yukw777/leela-zero-pytorch/blob/master/leela_zero_pytorch/network.py

 But this is strange in Resnet towers, because the convolutional bias never uses in Resnet towers. Seem to beta in towers will be use during the training time but ignoring in Leelaz.
Why?
